features/src/main/scala/com/salesforce/op/features/types/package.scala

```diff
@@ -196,6 +196,13 @@ package object types extends FeatureTypeSparkConverters {
   implicit class VectorConversions(val v: Vector) extends AnyVal {
     def toOPVector: OPVector = new OPVector(v)
   }
+  // Arrays
+  implicit class ArrayDoubleConversions(val v: Array[Double]) extends AnyVal {
+    def toReal: Seq[Real] = v.map(_.toReal)
```

Comment:
keep the Array type in return value, ie. `def toReal: Array[Real]` and `def toRealNN: Array[RealNN]`